css-brochure-form/styles.css

```diff
@@ -0,0 +1,97 @@
+html {
+  box-sizing: border-box;
+}
+
+form {
+  width: 800px;
+  margin: auto;
+  justify-content: center;
+}
+
+body {
+  background-color: rgb(255 255 255);
+  width: auto;
+  margin: auto;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```

Comment:
These two lines are not necessary, they are only needed in the `@font-face` rules we provided for you.